dpnp/dpnp_iface_statistics.py

```diff
@@ -169,6 +170,28 @@ def average(x1, axis=None, weights=None, returned=False):
     return call_origin(numpy.average, x1, axis, weights, returned)
 
 
+def bincount(x1, weights=None, minlength=0):
+    """
+    Count number of occurrences of each value in array of non-negative ints.
+
+    For full documentation refer to :obj:`numpy.bincount`.
+
+    See Also
+    --------
+    :obj:`dpnp.unique` : Find the unique elements of an array.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> x = np.bincount(np.arange(5))
+    >>> [i for i in x]
```

Comment:
```suggestion
    >>> print(x)
```
Do not create a new object just for printing